tfhe/src/high_level_api/integers/types/base.rs

```diff
@@ -919,31 +952,343 @@ generic_integer_impl_operation_assign!(RotateRightAssign(rotate_right_assign) =>
+        (super::FheUint64, u64),
+        (super::FheUint128, u128),
+        (super::FheUint256, U256),
+);
+generic_integer_impl_scalar_operation_assign!(
+    rust_trait: MulAssign(mul_assign),
+    key_method: scalar_mul_assign_parallelized,
+    fhe_and_scalar_type:
+        (super::FheUint8, u8),
+        (super::FheUint10, u16),
+        (super::FheUint12, u16),
+        (super::FheUint14, u16),
+        (super::FheUint16, u16),
+        (super::FheUint32, u32),
+        (super::FheUint64, u64),
+        (super::FheUint128, u128),
+        (super::FheUint256, U256),
+);
+generic_integer_impl_scalar_operation_assign!(
+    rust_trait: BitAndAssign(bitand_assign),
+    key_method: scalar_bitand_assign_parallelized,
+    fhe_and_scalar_type:
+        (super::FheUint8, u8),
+        (super::FheUint10, u16),
+        (super::FheUint12, u16),
+        (super::FheUint14, u16),
+        (super::FheUint16, u16),
+        (super::FheUint32, u32),
+        (super::FheUint64, u64),
+        (super::FheUint128, u128),
+        (super::FheUint256, U256),
+);
+generic_integer_impl_scalar_operation_assign!(
+    rust_trait: BitOrAssign(bitor_assign),
+    key_method: scalar_bitor_assign_parallelized,
+    fhe_and_scalar_type:
+        (super::FheUint8, u8),
+        (super::FheUint10, u16),
+        (super::FheUint12, u16),
+        (super::FheUint14, u16),
+        (super::FheUint16, u16),
+        (super::FheUint32, u32),
+        (super::FheUint64, u64),
+        (super::FheUint128, u128),
+        (super::FheUint256, U256),
+);
+generic_integer_impl_scalar_operation_assign!(
+    rust_trait: BitXorAssign(bitxor_assign),
+    key_method: scalar_bitxor_assign_parallelized,
+    fhe_and_scalar_type:
+        (super::FheUint8, u8),
+        (super::FheUint10, u16),
+        (super::FheUint12, u16),
+        (super::FheUint14, u16),
+        (super::FheUint16, u16),
+        (super::FheUint32, u32),
+        (super::FheUint64, u64),
+        (super::FheUint128, u128),
+        (super::FheUint256, U256),
+);
+generic_integer_impl_scalar_operation_assign!(
+    rust_trait: ShlAssign(shl_assign),
+    key_method: scalar_left_shift_assign_parallelized,
+    fhe_and_scalar_type:
+        (super::FheUint8, u8, u16, u32, u64, u128),
+        (super::FheUint10,u8, u16, u32, u64, u128),
+        (super::FheUint12, u8, u16, u32, u64, u128),
+        (super::FheUint14, u8, u16, u32, u64, u128),
+        (super::FheUint16, u8, u16, u32, u64, u128),
+        (super::FheUint32, u8, u16, u32, u64, u128),
+        (super::FheUint64, u8, u16, u32, u64, u128),
+        (super::FheUint128, u8, u16, u32, u64, u128),
+        (super::FheUint256, u8, u16, u32, u64, u128, U256),
+);
+generic_integer_impl_scalar_operation_assign!(
+    rust_trait: ShrAssign(shr_assign),
+    key_method: scalar_right_shift_assign_parallelized,
+    fhe_and_scalar_type:
+        (super::FheUint8, u8, u16, u32, u64, u128),
+        (super::FheUint10,u8, u16, u32, u64, u128),
```

Comment:
formatting for all FheUint10 lines is off, missing a space after the first comma

Reply:
fixed